src/main/java/net/mcreator/ui/modgui/LivingEntityGUI.java

```diff
@@ -974,6 +984,9 @@ private void regenerateAITasks() {
 		isBoss.setSelected(livingEntity.isBoss);
 		hasSpawnEgg.setSelected(livingEntity.hasSpawnEgg);
 		disableCollisions.setSelected(livingEntity.disableCollisions);
+		disableCollisions.setEnabled(!livingEntity.solidBoundingBox);

```

Comment:
Enabling and disabling is usually done in a method that is then called in needed places (init, listeners, edit mode data load). Check other ME UIs to see for examples on how we do this usually

Reply:
I am going to be changing this from a checkbox to a condition, and since that means it won't always be solid, I'll no longer need to disable it. (and as I mentioned before, enabling both didn't cause any problems)